words/objects.txt

```diff
@@ -838,14 +1071,19 @@ fork
 form
 fortnight
 fossa
+fosythia
```

Comment:
i think this is supposed to be spelled forsythia?

Reply:
it is!